dbt/models/reporting/reporting.ratio_stats.py

```diff
@@ -1,237 +1,234 @@
-# type: ignore
 # pylint: skip-file
-sc.addPyFile(  # noqa: F821
-    "s3://ccao-dbt-athena-dev-us-east-1/packages/spark-packages.zip"
-)
+def model(dbt, spark_session):
+    dbt.config(materialized="table")
 
-import numpy as np  # noqa: E402
-import pandas as pd  # noqa: E402
-from assesspy import boot_ci  # noqa: E402
-from assesspy import cod  # noqa: E402
-from assesspy import prd_met  # noqa: E402
-from assesspy import cod_ci as cod_boot  # noqa: E402
-from assesspy import cod_met, mki, mki_met, prb, prb_met, prd  # noqa: E402
-from assesspy import prd_ci as prd_boot  # noqa: E402
+    # Load dependency bundle so that we can import deps
+    python_model_dependency = dbt.ref("ccao.python_model_dependency")
+    s3_dependency_dir = python_model_dependency.first()["s3_dependency_dir"]
```

Comment:
One major downside of this solution is that the `sc.addPyFile()` call and any subsequent `import` calls must be defined inside the context of the `model` function, since `sc.addPyFile()` won't know where to load the bundle from until we can pull `s3_dependency_dir` from the `ccao.python_model_dependency` table using a `dbt.ref()` call. This is slightly annoying insofar as it makes the Python model file look less similar to a regular Python script than it would otherwise be, but I don't actually think it's a particularly big deal.